pulsar-client-cpp/README.md

```diff
@@ -202,8 +202,7 @@ ${PULSAR_PATH}/pulsar-client-cpp/tests/
 
 # Execution
 # Start standalone broker
-export PULSAR_STANDALONE_CONF=${PULSAR_PATH}/pulsar-client-cpp/tests/standalone.conf
-${PULSAR_PATH}/bin/pulsar standalone
+${PULSAR_PATH}/pulsar-test-service-start.sh
```

Comment:
We could also mention `pulsar-test-service-stop.sh` to tear down the service

Reply:
Added, and rebased to master and repushed.